lib/pf/scan.pm

```diff
@@ -145,8 +147,9 @@ sub parse_scan_report {
                'mac' => $mac,
                'reason' => 'manage_vclose',
             );
+        
             $apiclient->notify('close_security_event', %data );
-            $apiclient->notify('reevaluate_access', %data );
+
```

Comment:
No need to reevaluate access after parsing of the report , reevaluate should be in the event actions .